WIP Added buttons to user profile page for GH/OSO link/unlink

sbryzak opened the pull request:
### Work in progress

This PR adds buttons to the user profile page, allowing the connected GitHub and Openshift accounts to be (re)linked/unlinked.  See this issue for more details:

[](https://github.com/fabric8-services/fabric8-auth/issues/151)

Some styling is still required to position buttons correctly.

These changes depend on the following PR in the ngx-login-client module:

[](https://github.com/fabric8-ui/ngx-login-client/pull/94)

Signed-off-by: Shane Bryzak <sbryzak@redhat.com>

mindreeper2420 commented on src/app/profile/overview/connected-accounts/connected-accounts.component.html:
```diff
@@ -41,10 +41,20 @@ <h2 class="card-pf-title">
                         <span>
                           Connected
                         </span>
+                        <span>
```
you can remove these `<span>` tags

mindreeper2420 commented on src/app/profile/overview/connected-accounts/connected-accounts.component.html:
```diff
@@ -41,10 +41,20 @@ <h2 class="card-pf-title">
-                          Not Connected
+                          Disconnected
+                        </span>
+                        <span>
```
can also be removed

mindreeper2420 commented on src/app/profile/overview/connected-accounts/connected-accounts.component.html:
```diff
@@ -77,11 +87,21 @@ <h2 class="card-pf-title">
                         <span>
                           {{contextUserName}}
                         </span>
+                        <span>
```
remove as well

mindreeper2420 commented on src/app/profile/overview/connected-accounts/connected-accounts.component.html:
```diff
@@ -77,11 +87,21 @@ <h2 class="card-pf-title">
                         <span class="account-disconnected">
                           Not Connected
                         </span>
+                        <span>
```
last one to remove

mindreeper2420 commented on src/app/profile/overview/connected-accounts/connected-accounts.component.html:
```diff
@@ -41,10 +41,20 @@ <h2 class="card-pf-title">
                         <span>
```
add class `margin-right-15` here to add a nice space between this text and the button group.

mindreeper2420 commented on src/app/profile/overview/connected-accounts/connected-accounts.component.html:
```diff
@@ -41,10 +41,20 @@ <h2 class="card-pf-title">
+                        </span>
                       </ng-template>
                       <ng-template #ghUnlinkedUsername>
                         <span class="account-disconnected">
```
same here - add class `margin-right-15` to apply a space between this text and the button

mindreeper2420 commented on src/app/profile/overview/connected-accounts/connected-accounts.component.html:
```diff
@@ -77,11 +87,21 @@ <h2 class="card-pf-title">
                         <span>
```
add `class="margin-right-15"`

mindreeper2420 commented on src/app/profile/overview/connected-accounts/connected-accounts.component.html:
```diff
@@ -77,11 +87,21 @@ <h2 class="card-pf-title">
+                        </span>
                       </ng-template>
                       <ng-template #osoUnlinkedUsername>
                         <span class="account-disconnected">
```
add `class="margin-right-15"`

sbryzak commented:
I've updated the button styles, and removed the extraneous <span> elements.  I've also added the margin-right-15 class to the "reconnect" buttons in order to place a gap between the two buttons.  I'm still working on updating the UI upon a successful disconnect of either accounts, however I've been stuck for quite a few hours on getting this to happen in the authentication service.

joshuawilson commented:
please rebase

joshuawilson commented:
@sbryzak what is the status of this?

alexeykazakov commented:
We can close this. @dgutride is working on this.
